lib/expand/join.go

```diff
@@ -725,6 +725,7 @@ func (p *Peer) runLocalChecks(ctx operationContext) error {
 			VxlanPort: int32(installOperation.GetVars().OnPrem.VxlanPort),
 			DnsAddrs:  ctx.Cluster.DNSConfig.Addrs,
 			DnsPort:   int32(ctx.Cluster.DNSConfig.Port),
+			Openebs:   ctx.Cluster.App.Manifest.OpenEBSEnabled(),
```

Comment:
Nit: This field is generated from the proto file, right? Just curious if there's a way to make it `OpenEBS` (with capital abbreviation) so it's consistent. Don't worry about it if it's not trivial.